bot/utils/parameter_value_generator.py

```diff
@@ -14,6 +14,9 @@ def __init__(self, fun, parameter):
     def __call__(self, *args):
         return self.parameter * self.fun(*args)
 
+    def __str__(self):
+        return str(self.parameter)
```

Comment:
will it be better to return something like `"* " + str(self.parameter)` to let everyone know that the function is being scaled by this factor?

Reply:
Yeah. I wonder if you can get the print name of the function from what @priyanshuone6 has been doing? E.g. for varying a diffusivity function it would be cool to print `D * scale`

Reply:
I discussed this with @priyanshuone6 but I couldn't figure out a way to print the name of a particular parameter from the parameters dictionary. Should I add them manually (`if-else` condition in `__str__`)? As of right now, the bot is varying only 2 functional parameters - `"Positive electrode exchange-current density [A.m-2]"` and `"Negative electrode exchange-current density [A.m-2]"`.

I also tried doing this -
```py
model = pybamm.lithium_ion.DFN()
print(model.variables["Negative electrode exchange current density [A.m-2]"].to_equation())
```
which gives me -
```
current_scale*j0_n/(C_r_n*L_x*\hat{j0}_{n}^{ref}*a_{n}^{typ})
```
but I don't think that this is the same as the name of `parameters["Negative electrode exchange-current density [A.m-2]"]`

Reply:
to_equation() parses the tree and gives you what the variable is. Doing
`print(model.variables["Negative electrode exchange current density [A.m-2]"]. print_name)` will return the symbol for the variable

Currently the print name for all the model variables is `None` unless manually set.
You can manually set the print_name like for Sum of interfacial current densities in full_conductivity.py you can do
```python
sum_j = variables["Sum of interfacial current densities"]
sum_j.print_name = "J" 
```

Reply:
Oh yes, I forgot to mention that I did try `print_name` which gave `None` as the output as it isn't set. But I am still not sure if setting up the name will help in this case.